bundles/AdminBundle/Resources/public/js/pimcore/document/editables/areablock.js

```diff
@@ -1056,6 +1057,22 @@ pimcore.document.editables.areablock = Class.create(pimcore.document.area_abstra
                 items: buttons
             });
 
+            Ext.on('resize', function() {
+                var winH = 0;
+                if (document.body && document.body.offsetHeight) {
+                    winH = document.body.offsetHeight;
+                }
+                if (document.compatMode=='CSS1Compat' && document.documentElement && document.documentElement.offsetHeight ) {
+                    winH = document.documentElement.offsetHeight;
+                }
+                if (window.innerHeight) {
+                    winH = window.innerHeight;
+                }   
```

Comment:
Should these be if / else if / else if?

Reply:
I could probably write it that way but Its meant to determine what API the current browser is using to provide document height and widths.